website/settings/defaults.py

```diff
@@ -158,7 +158,8 @@ def parent_dir(path):
 SHORT_DOMAIN = 'osf.io'
 
 # TODO: Combine Python and JavaScript config
-COMMENT_MAXLENGTH = 500
+# Add 500 to account for variable length due to at mentions
+COMMENT_MAXLENGTH = 500 + 500
```

Comment:
Yeah, and it's [using](https://github.com/leb2dg/osf.io/blob/6d3210ceead6e71d341f19f5399a226bd547a4e6/website/static/js/koHelpers.js#L482) `innerText` for the comparison, which is why we can use the 500 character limit directly there. On the server side we'd need to (re)strip the HTML in order to validate it against the 500 character limit. Don't we have `bleach` installed? Can we use that?